### DISCUSSION TITLE
Request to validate the Hashmod config

### DISCUSSION THREAD

**Discussion by velu4689** [prometheus/prometheus#14110] [Q&A]
### What did you do?

We have used following hashmod config in our prometheus (on prometheus operator with horizontally scalable)

In relabel_config, we have used the following:  replicas (2), shards (3)
      - source_labels: [__meta_kubernetes_pod_label_nfId]
        modulus: 3
        target_label:  __tmp_hash
        action: hashmod
      - source_labels: [__tmp_hash]
        regex: ^1$
        action: keep

However, we are unable to add get the data for certain hosts. However we receive data when we use the following scrape config, replicas (1), shards (3)
      - source_labels: [__meta_kubernetes_pod_label_nfId]
        modulus: 3
        target_label:  __tmp_hash
        action: hashmod

Kindly review & let me know if this is correct. Also, if second config is really valid.

### What did you expect to see?

Data from all hosts

### What did you see instead? Under which circumstances?

No data

### System information

Linux

**Comment by machine424** [comment]
This looks like a support request, I'll convert this into a discussion.
Don't hesitate to reach out to the community over slack https://prometheus.io/community/ (there is more activity there)
